sdk/python/kfp/_client.py

```diff
@@ -262,7 +262,7 @@ def _get_url_prefix(self):
         return 'http://' + self._uihost
 
     # In-cluster pod. We could use relative URL.
-    return '/pipeline'
+    return '/_/pipeline'
```

Comment:
This is a problem between "KFP UI in Kubeflow centraldashboard" vs "standalone KFP UI". In multi user mode, KFP UI does not work standalone, so we need to add this `/_/` thing.

When you make the change, it will be broken for KFP standalone too. So I think a proper fix should be letting KFP API server expose information about the correct setup and construct the URL according to information fetched from that endpoint: https://github.com/kubeflow/pipelines/issues/4637.

Reply:
@Bobgy this shouldn't be an issue anymore since now the shared lib provided by centraldashboard will take care of adding the missing `/_/` if needed. See https://github.com/kubeflow/kubeflow/pull/5469

Reply:
Awesome, that seems like a better solution